templates/csharp/custom-copilot-rag-api/{{ProjectName}}.csproj.tpl

```diff
@@ -0,0 +1,74 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>{{TargetFramework}}</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+  </PropertyGroup>
+
+{{^isNewProjectTypeEnabled}}
+  <ItemGroup>
+    <ProjectCapability Include="TeamsFx" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Include="appPackage/**/*" />
+    <None Include="infra/**/*" />
+    <None Remove="devTools/**" />
+    <Content Remove="devTools/**/*" />
+    <None Include="env/**/*" />
+  </ItemGroup>
+
+{{/isNewProjectTypeEnabled}}
+  <ItemGroup>
+    <PackageReference Include="AdaptiveCards" Version="3.1.0" />
+    <PackageReference Include="AdaptiveCards.Templating" Version="1.5.0" />
+    <PackageReference Include="Microsoft.Bot.Builder" Version="4.22.7" />
+    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.22.7" />
+    <PackageReference Include="Microsoft.OpenApi" Version="1.6.19" />
+    <PackageReference Include="Microsoft.OpenApi.Readers" Version="1.6.19" />
+    <PackageReference Include="Microsoft.Teams.AI" Version="1.5.*" />
+    <PackageReference Include="RestSharp" Version="112.0.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="Prompts\chat\skprompt.txt">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
+    </Content>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="Prompts\chat\actions.json">
```

Comment:
It seems json file already included in Content Group. Not necessary to add them here.

Reply:
Sure, will fix it